packages/storage-plugin/src/internals.ts

```diff
@@ -53,8 +53,8 @@ export function storageOptionsFactory(
   return {
     key: [DEFAULT_STATE_KEY],
     storage: StorageOption.LocalStorage,
-    serialize: JSON.stringify,
-    deserialize: JSON.parse,
+    serialize: obj => JSON.stringify(obj),
+    deserialize: obj => JSON.parse(obj),
```

Comment:
Once you have done the recommendation below please see if you can revert these changes and it is happy with it

Reply:
@davidmorissette see